src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandResult.java

```diff
@@ -47,15 +47,17 @@ public record CommandResult(
                   nullable = true)
             })
         Map<CommandStatus, Object> status,
-    @JsonInclude(JsonInclude.Include.NON_EMPTY) @Schema(nullable = true) List<Error> errors) {
+    @JsonInclude(JsonInclude.Include.NON_EMPTY) @Schema(nullable = true) List<Error> errors,
+    @Schema(description = "Deprecated command warning message.", nullable = true)
```

Comment:
This probably needs to be discussed, whether we want another main-level property instead of a new entry within "status"

Reply:
Thanks, that should be discussed. @amorton

Reply:
please refer to the comment from Aaron in this PR.